### PR TITLE
Adding payment_notes

### PR DESCRIPTION
**Pull request by ChujiaGuo**

### CODE REVIEW

File: constellations/orion/src/tests_integration/class_integration_test.go [hunk 1]
```diff
@@ -195,6 +195,7 @@ func createClass(id int) domains.Class {
 			StartDate:       now,
 			EndDate:         later1,
 			GoogleClassCode: domains.NewNullString("ab12cd34"),
+			PaymentNotes:    domains.NewNullString("notes1"),
```
Comment (ahsu1230) [inline constellations/orion/src/tests_integration/class_integration_test.go:198]: Can you test Null notes somewhere in these tests? Doesn't have to be here.

Comment (ahsu1230) [reply constellations/orion/src/tests_integration/class_integration_test.go:198]: I meant test "" string for `notes`.

Comment (ChujiaGuo) [reply constellations/orion/src/tests_integration/class_integration_test.go:198]: oh i did, i guess i just didn't push it that way...

File: constellations/orion/src/tests_integration/class_integration_test.go [hunk 2]
```diff
@@ -300,6 +304,7 @@ func assertClass(t *testing.T, id int, class domains.Class) {
 		assert.EqualValues(t, 0, class.FullState)
 		assert.EqualValues(t, 0, class.PricePerSession.Uint)
 		assert.EqualValues(t, 10, class.PriceLump.Uint)
+		assert.EqualValues(t, "", class.PaymentNotes.String)
```
Comment (ahsu1230) [inline constellations/orion/src/tests_integration/class_integration_test.go:307]: woohoo awesome!

**Pull request merged**